test/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Testing file for eslint rules
```

Comment:
What's the point of this file?

Reply:
Only for testing the tslint configuration inside the pipeline as `lint` step. If is useless we can remove this file and the lint step from the pipeline.

Reply:
Ok, so for this repository the `lint` step serves as the `test` step 🙃

I like the idea. I just really make it explicit by renaming the `lint` script into `test`, so
```json
"test": "eslint . -c .eslintrc.js --ext .ts,.tsx"
```
We're actually not linting linters <img src="http://www.gomotes.com/emoticon/watchmensmiley.gif" alt="Watchmen Smiley" />

Also, with this file we can only test lint positiveness. How can we test our rules catch what we consider to be errors? (not sure it's possibile)

Reply:
removed in 368b192

Reply:
We can explore some testing scenario on a proper PR. For now I'm removing the testing file.

Reply:
Should we remove this file as well https://github.com/pagopa/eslint-rules/blob/npm-release-pipeline/.eslintrc.js?
And the lint script of course 

(ok I'm nitpicking)

Reply:
removing useless files and lint script in 082ac50

Reply:
> We can explore some testing scenario on a proper PR. For now I'm removing the testing file.

What we can actually do is create a file with positive examples and one for negatives. Let's say `test.ok.ts` and `test.ko.js`.

Then we can implement the following script: 
```sh
testOK=$(eslint test.ok.ts) && echo 0 || echo 1 
testKO=$(eslint test.ko.ts) && echo 1 || echo 0
[$testOK == 1] && [$testKO == 1] && exit 0 || exit 1
```
(ok idk if it actually works, but you have the point).

We can set ti up in another PR, tough.